### PR TITLE
Test cases for multlple addresses

### PR DESCRIPTION
**Pull request by fords**
Adding test cases

### CODE REVIEW

File: mycity/mycity/intents/trash_intent.py [hunk 1]
```diff
@@ -153,23 +153,19 @@ def get_trash_and_recycling_days(address, neighborhood=None):
...
     :param address_request_json: json object returned from ReCollect address
         request service
-    :return: list of unique addresses
+    :return: list of one unique address
     """
     logger.debug('address_request_json: ' + str(address_request_json))
     # Pre-extract the addresses from the payload and uniquify them
-    strings_to_compare = sorted(set(address["name"] for address in address_request_json), key=len, reverse=True)
+    # strings_to_compare = sorted(set(address["name"] for address in address_request_json), key=len, reverse=True)
+    return [address_request_json[0]["name"]]
```
Comment (hamstap85) [inline mycity/mycity/intents/trash_intent.py]: I can add documentation why we're using the first address here

Comment (fords) [reply mycity/mycity/intents/trash_intent.py]: That'd be great.

Comment (fords) [reply mycity/mycity/intents/trash_intent.py]: Thanks for the help. Happy new year

**Comment by Joshhw** [comment]
Closing PR cause its too old at this point.

**Pull request closed without merging**